hazelcast/src/main/java/com/hazelcast/internal/management/dto/SlowOperationDTO.java

```diff
@@ -0,0 +1,74 @@
+package com.hazelcast.internal.management.dto;
+
+import com.eclipsesource.json.JsonArray;
+import com.eclipsesource.json.JsonObject;
+import com.eclipsesource.json.JsonValue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.hazelcast.util.JsonUtil.getArray;
+import static com.hazelcast.util.JsonUtil.getInt;
+import static com.hazelcast.util.JsonUtil.getString;
+
+/**
+ * A Serializable DTO for {@link com.hazelcast.spi.impl.operationexecutor.slowoperationdetector.SlowOperationLog}.
+ */
+public class SlowOperationDTO {
```

Comment:
Those DTOs are not (de)serialized by themselves. This is done exclusively via `LocalOperationStatsImpl`. So the `fromJSON()` method will never be used.